From ebfa088029b1450c99154db2ae5bd27936dde54d Mon Sep 17 00:00:00 2001
From: Debarshi Ray <debarshir@gnome.org>
Date: Tue, 16 Feb 2021 21:41:04 +0100
Subject: [PATCH] domain-ontology-rule: Use private locations for the cache &
 the journal

GNOME Photos uses private instances of the Tracker 2.x daemons. Using
the global location for the Tracker cache and the journal can cause
contention with other instances of Tracker running on the system,
leading to unknown behaviour.

Therefore, it's better to use private locations for private Tracker
instances.

https://gitlab.gnome.org/GNOME/gnome-photos/-/merge_requests/162
---
 data/org.gnome.Photos.rule.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/org.gnome.Photos.rule.in b/data/org.gnome.Photos.rule.in
index 3d9c2bb3..e2d2bdfe 100644
--- a/data/org.gnome.Photos.rule.in
+++ b/data/org.gnome.Photos.rule.in
@@ -14,11 +14,11 @@
 #
 
 # Location for the Tracker database (not optional)
-CacheLocation=$XDG_CACHE_HOME/tracker
+CacheLocation=$XDG_CACHE_HOME/gnome-photos/tracker
 
 # Location for the data journal, used in data recovery
 # situations (optional)
-JournalLocation=$XDG_DATA_HOME/tracker/data
+JournalLocation=$XDG_DATA_HOME/gnome-photos/tracker/data
 
 # Name of the ontology to use, must be one located in
 # $(sharedir)/tracker/ontologies
-- 
GitLab

